### PR TITLE
split up report header lines for config, rootdir and testpaths

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
i found it painful to read crammed in a single line thus rootdir, config file and testpaths now have own lines

- [x] basic tests
- [x] changelog
- [x] regendoc (skipped as there seem to be more changes i didn't intend)

### CODE REVIEW

File: doc/en/example/simple.rst [hunk 1]
```diff
@@ -951,46 +951,10 @@ and run it:
...
-
-    test_module.py:19: AssertionError
-    ========================= short test summary info ==========================
-    FAILED test_module.py::test_call_fails - assert 0
-    FAILED test_module.py::test_fail2 - assert 0
-    ERROR test_module.py::test_setup_fails - assert 0
-    ======================== 2 failed, 1 error in 0.12s ========================
+    ImportError while loading conftest '/home/sweet/project/conftest.py'.
+    conftest.py:4: in <module>
+        phase_report_key = StashKey[Dict[str, CollectReport]]()
```
Comment (nicoddemus) [inline doc/en/example/simple.rst:956]: This seems wrong.

**Review by nicoddemus — APPROVED** [review]
LGTM! Just please take at my comment.

File: doc/en/example/simple.rst (hunk 1 above)
Comment (RonnyPfannschmidt) [reply doc/en/example/simple.rst]: was a preexisting bug

File: changelog/10727.improvement.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Split the report header for ``rootdir``, ``config file`` and ``testpaths`` so each has own line.
```
[the remark this answers is not in the document]
Comment (RonnyPfannschmidt) [reply changelog/10727.improvement.rst]: fixed, thanks

**Pull request merged**